### DISCUSSION TITLE
The lock file maintenance PR locked by rate limit

### DISCUSSION THREAD

**Discussion by frbuceta** [renovatebot/renovate#9041] [Request Help]
Hello, I come here to comment on a case that is occurring in my organization with regard to lock files that cannot be merged due to the rate limit.

The renovate configuration file is here (https://github.com/strongloop/loopback-next/blob/master/.github/renovate.json).
The dependencies panel I also point it here (https://github.com/strongloop/loopback-next/issues/3042)

As you can see, right now the PR is blocked by the rate limit. Any ideas to make it an exception? Or that scheduled tasks skip the rate limit?

**Comment by rarkins** [comment]
If you add `"prConcurrentLimit": 0` inside the `lockFileMaintenance` object in config then they should exclude it from concurrent PR rate limiting.

**Reply by frbuceta** [reply]
OK thank you so much

**Reply by frbuceta** [reply]
@rarkins I just observed that I still have the same problem using this.

See

- https://github.com/strongloop/loopback-next/blob/50db3c43a07e8ed7ef85332e2da490cd89785e93/.github/renovate.json5#L30-L37

- https://github.com/strongloop/loopback-next/issues/3042

I leave an image because I suppose it will end up merging now that I have removed a dependency
![Screenshot 2021-04-14 at 09 18 46](https://user-images.githubusercontent.com/13822438/114669791-95950980-9d02-11eb-8707-35fdefb5aef0.png)

**Reply by rarkins** [reply]
Sorry, it appears I was wrong. `prConcurrentLimit` is not calculated per-branch.

**Comment by rarkins** [comment]
If you check the dashboard option, does it open?

**Reply by frbuceta** [reply]
Yes. And is there any solution to fix this?

**Reply by rarkins** [reply]
Not currently. PrConcurrentLimit applies to all PRs equally.

**Reply by frbuceta** [reply]
Can we modify something in the renovate code to look for this functionality?

**Reply by rarkins** [reply]
It could be done, but it's not simple. If you're interested to contribute the feature, please create a feature request issue first and we can discuss what is the best way to approach it.
